README.md

```diff
@@ -75,7 +79,12 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+var pizza = {
+  temperature: 70,
+  bake: function(){
+    temperature = 300;
+  }
+}
 ```
```

Comment:
When setting temperature to a new value, so you need to reference it using `this.temperature`.